@noun-auction/typings/index.ts

```diff
@@ -66,31 +33,78 @@ export type DaoConfigProps = {
   abi: any
 }
 
-export type ContractAuctionData = {
-  auction:
-    | {
-        nounId: string
-        amount: string
+export type TokenPrice = {
+  decimal: number
+  raw: string
+}
+
+export type ActiveNounishAuctionResponse =
+  | {
+      /**
+       * Nounish NFT address
+       */
+      collectionAddress: string
+      /**
+       * Nounish Auction Contract Address
+       */
+      marketAddress: string
+      /**
+       * Zora API Market Type Classifier
+       */
+      marketType: 'ACTIVE_LIL_NOUNS_AUCTION' | 'ACTIVE_NOUNS_AUCTION'
+      properties: {
+        /**
+         * Auction ID = Token ID
+         */
+        auctionId: string
+        /**
+         * ID of Token/NFT up for auction
+         */
+        tokenId: string
+        /**
+         * Currency used for AUCTION. Curreny ETH (0 Address)
+         */
+        auctionCurrency: string
+        /**
+         * Unix TimeStamp
+         */
         startTime: string
+        /**
+         * Unix TimeStamp
+         */
         endTime: string
-        bidder: string
-        settled: boolean
+        /**
+         * Numerical String
+         */
+        duration: string
+        /**
+         * Date object: 2022-08-15T17:33:54+00:00
+         */
+        estimatedDurationTime: string
+        /**
+         * Date object: 2022-08-15T17:33:54+00:00
+         */
+        firstBidTime: string
+        /**
+         * null - should be numerical string
+         */
+        minBidIncrementPercentage: null
+        /**
+         * null?
+         */
+        timeBuffer: null
+        /**
+         * ETH Wallet Address
+         */
+        highestBidder: string | null
+        /**
+         * ETH Wallet Address
+         */
+        winner: string | null
+        highestBidPrice: {
+          chainTokenPrice: TokenPrice | null
+          usdcPrice: TokenPrice | null
+        }
       }
-    | undefined
-}
-
-export type NormalizedAuctionData = {
-  countdown: {
-    startTime: string
-    endTime: string
-  }
-  highBid: {
-    ethValue: string
-    usdcValue: string
-  }
-  bidder: {
-    address: string
-    txHash: string
-  }
-  rpcData: ContractAuctionData
-}
+    }
+  | undefined
```

Comment:
yes, I agree it kinda belong somewhere on library level, other than app level

Reply:
Yeah, it feels a little funny to have to hardcode the various classifiers etc... but it does illustrate the more specific data points available to those who care to look into the codebase.